be/src/service/internal_service.cpp

```diff
@@ -391,7 +391,10 @@ void PInternalServiceImpl::open_load_stream(google::protobuf::RpcController* con
         }
 
         stream_options.handler = load_stream.get();
-        stream_options.idle_timeout_ms = config::load_stream_idle_timeout_ms;
+        stream_options.idle_timeout_ms = request->idle_timeout_ms();
+        DBUG_EXECUTE_IF("LoadStreamStub._send_with_retry.delay_before_send", {
```

Comment:
Maybe use another debug point name? this one is already taken by LoadStreamStub::_send_with_retry.